crates/test_utils/src/main.rs

```diff
@@ -26,6 +26,9 @@ struct Args {
     /// Admin API Key of the environment
     #[arg(short, long = "admin_api_key")]
     admin_api_key: String,
+    /// Optional Verbose logs
+    #[arg(short, long = "verbose")]
```

Comment:
```suggestion
    #[arg(short, long)]
```

This should just work.